scylladb/sphinx-scylladb-theme
# Add a collapsible component

Closes https://github.com/scylladb/sphinx-scylladb-theme/issues/276


## How to test this PR

1. Clone this PR. For more information, see [Cloning pull requests locally](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally).

2. Enter the docs folder, and run:

```
make preview
````

3. Open http://127.0.0.1:5500/examples/collapse/ with your favorite browser

4. Test the new component:

![image](https://user-images.githubusercontent.com/9107969/150154739-eb129796-e8f4-46b3-8b6c-eed4f70577f2.png)
